tests/integration/upload_file_tests/test_upload_file.py

```diff
@@ -0,0 +1,73 @@
+from tests.integration.base import DBTIntegrationTest, use_profile
+import yaml
+
+
+class TestUploadFileCSV(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return "adapter_methods_caching"
```

Comment:
I might suggest a name change on here and lines 32 and 56 to something like 'upload_file'.

Reply:
Good spot! Done, 1f03a041047e6820c6d717807dbf5b9111a6104b.